### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by ekoi**

### CODE REVIEW

File: docker-compose-local-arm.yml
Comment (kerim1) [inline docker-compose-local-arm.yml:1]: I have already updated the main `docker-compose.yml` file. So this one is not necessary anymore.

File: registry/frontend/src/components/ReportAbuse.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import * as Dialog from '@radix-ui/react-dialog';
+import { Cross2Icon } from '@radix-ui/react-icons';
+import './styles.css'
+import {Text, TextArea} from "@radix-ui/themes";
+import * as Checkbox from '@radix-ui/react-checkbox';
+import { CheckIcon } from '@radix-ui/react-icons';
+
+const DialogDemo = () =>{
```
Comment (kerim1) [inline registry/frontend/src/components/ReportAbuse.tsx:9]: Guessing this is still work in progress? As it is called `DialogDemo` and not `ReportAbuse`?

File: registry/frontend/src/components/ReportAbuse.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    <Dialog.Trigger asChild>
+      <button className="Button">Report Abuse</button>
+    </Dialog.Trigger>
+    <Dialog.Portal>
+      <Dialog.Overlay className="DialogOverlay" />
+      <Dialog.Content className="DialogContent">
+         <form
+            onSubmit={async (event) => {
+                const data = Object.fromEntries(new FormData(event.currentTarget));
+                const url_target = "/mail"
```
Comment (kerim1) [inline registry/frontend/src/components/ReportAbuse.tsx:24]: The url target should come as a property to the `ReportAbuse` component.

File: registry/frontend/src/components/ReportAbuse.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                        "subject": subject
+
+                    })
+
+
+                }).then(response => {
+                    console.log(response.status)
+                })
+
+                wait().then(() => setOpen(false));
```
Comment (kerim1) [inline registry/frontend/src/components/ReportAbuse.tsx:57]: Why the `wait`?

File: registry/frontend/src/components/ReviewForm.tsx
Comment (kerim1) [inline registry/frontend/src/components/ReviewForm.tsx:1]: File can be deleted? As it is now in the review-component instead.

File: registry/frontend/src/components/Reviews.tsx [hunk 4]
```diff
@@ -1,28 +1,40 @@
 import {Vocab} from '../misc/interfaces';
+import ReviewForm from "./ReviewForm";
+import StarRating from "./StarRating";
```
Comment (kerim1) [inline registry/frontend/src/components/Reviews.tsx:3]: This should now come from the review-component.

File: registry/frontend/src/components/StarRating.tsx
Comment (kerim1) [inline registry/frontend/src/components/StarRating.tsx:1]: Can also be deleted as it is now in the review-component.

File: registry/frontend/src/components/Reviews.tsx [hunk 5]
```diff
@@ -1,28 +1,40 @@
...
+    const login = () => {
+       window.location.href = '/login'
+    };
     return (
         <>
             {data.reviews.map(review => (
                 <div className="review" key={review.id}>
                     <div className="hcAlignLeft hcMarginBottom1">
-                        <StarRating rating={review.rating}/>
+                        <div>
```
Comment (kerim1) [inline registry/frontend/src/components/Reviews.tsx:22]: Is the `<div>` really necessary?

**Review by kerim1 — CHANGES_REQUESTED** [review]
Some general comments here and there. Let's discuss this further on Monday!

File: registry/frontend/src/components/ReportAbuse.tsx (hunk 1 above)
Comment (ekoi) [reply registry/frontend/src/components/ReportAbuse.tsx:9]: Yep

**Pull request merged**